llvm/test/Instrumentation/MemorySanitizer/ARM32/vararg-arm32.ll

```diff
@@ -0,0 +1,91 @@
+; RUN: opt < %s -S -passes=msan 2>&1 -msan-origin-base=0x40000000 -msan-and-mask=0x80000000 | FileCheck %s
+
+target datalayout = "E-m:m-i8:8:32-i16:16:32-i32:32-n32-S64"
+target triple = "arm--linux"
+
+define i32 @foo(i32 %guard, ...) {
```

Comment:
Assuming this one was copied from something else, 
I suggest this sequence of pull requests:
1. Move/copy files unchanged
2. switch "target tripple" to 32bit and update tests with invalid instrumentation (this step only if compiled does not crash or fail on assert)
3. This PR #109284
 a PR with moving copying files unchanged, and in the current PR "target triple"

Reply:
Thank you.
Yes, this was copied from mips64 and modified to fit the tests.
So, the first step is we open a PR with tests for all the 32bit platforms, but their "target triple" is their respective 64bit counterpart? Did I understand it right?

Reply:
Second step is also unclear:
If we just switch the target triple to 32bit, most of the checks will be invalid. Thus, "update tests with invalid instrumentation" would mean update all tests.
Also, since this PR is the last, MemorySanitizer will not be updated, so when running tests between 2nd and 3rd PR's the tests will fail with "unsupported architecture" assert.
I believe, this is not the intended course of action. Could you please, elaborate on the steps?